public/413_logo.html

```diff
@@ -0,0 +1,121 @@
+                  <path d="M247.59,5.72 C239.24,5.72 232.92,11.61 232.92,19.57 C232.92,27.53 239.24,33.5 247.59,33.5 C255.94,33.5 262.26,27.5 262.26,19.57 C262.26,11.64 255.94,5.72 247.59,5.72 L247.59,5.72 Z M247.67,28.11 C242.984983,28.0769715 239.212292,24.2550511 239.24,19.57 C239.221275,17.3248681 240.102128,15.1656799 241.685969,13.5743235 C243.26981,11.9829671 245.424805,11.0919033 247.67,11.1 C252.369943,11.1000003 256.179999,14.910057 256.179999,19.61 C256.179999,24.309943 252.369943,28.1199997 247.67,28.12 L247.67,28.11 Z" id="Shape" fill="#FFFFFF"></path>
+                  <polygon id="Shape" fill="#FFFFFF" points="286.69 5.92 279.2 26.52 271.75 5.92 265.12 5.92 275.85 33.23 282.21 33.23 293.05 5.92"></polygon>
+                  <path d="M12.33,26 L13.78,17.72 C12.2933959,16.5679951 11.7057535,14.5970193 12.3187809,12.8190135 C12.9318083,11.0410078 14.6093016,9.85101384 16.49,9.86 L16.49,0.67 L5.29,0.67 C4.13896043,0.67 3.03506445,1.12724869 2.22115657,1.94115657 C1.40724869,2.75506445 0.95,3.85896043 0.95,5.01 L0.95,26.29 C0.946641375,26.6914806 1.03559765,27.0883624 1.21,27.45 C2.15,29.35 5.86,35.45 16.48,40.14 L16.48,26.34 C15.0894867,26.3456136 13.7010574,26.2318628 12.33,26 L12.33,26 Z" id="Shape" fill="#E21D3E"></path>
+                  <path d="M27.66,0.64 L16.48,0.64 L16.48,9.83 L16.48,9.83 C18.3520786,9.82031421 20.0243225,10.9988953 20.6446816,12.7652268 C21.2650407,14.5315582 20.6970038,16.4969557 19.23,17.66 L20.68,25.98 C19.2930776,26.2214841 17.8877871,26.3419376 16.48,26.34 L16.48,40.12 C27.09,35.45 30.81,29.33 31.75,27.43 C31.9244023,27.0683624 32.0133586,26.6714806 32.01,26.27 L32.01,5 C32.0153314,3.84376397 31.5590627,2.73319949 30.7424109,1.91467032 C29.9257591,1.09614115 28.8162452,0.637323542 27.66,0.64 L27.66,0.64 Z" id="Shape" fill="#B51E23"></path>
+                  </g>
+                  </g>
+                  </g>
+                </svg>
+              </div>
+            </div>
+          </div>
+          <div class="inner cover">
+            <h1>We're sorry, but your request was too large.</h1>
+            <p>
+            If you were trying to upload a logo for your app, please review our
+            <a href="https://developers.login.gov/design-guidelines/#agency-logo-guidelines" target="_blank">guidelines</a>
+            and make sure the image is no larger than 1MB. You can also go back
```

Comment:
Do we have any JS that can detect the file size before we submit the form to the server?

Reply:
I think that's the next step - right now we're just trying to mitigate the extremely unhelpful 413 error page that nginx throws when you upload a logo that's too big. The other option is to enable multi-part uploads to mitigate the performance and DDOS risk of large request sizes, but we wanted to try and get something quick and low-risk done first.